.devcontainer/devcontainer.json

```diff
@@ -1,5 +1,6 @@
 {
-  "image": "mcr.microsoft.com/vscode/devcontainers/base:ubuntu-24.04",
+  // TODO: Bump to ubuntu-24.04 once Python issues are fixed: https://github.com/argoproj/argo-workflows/pull/14302#issuecomment-2742883052
```

Comment:
In case anyone is unaware: this shouldn't cause any JSON parsing issues because comments are explicitly allowed by the spec: https://containers.dev/implementors/spec/